_posts/2023-06-20-introducing-iam-for-openfaas.md

```diff
@@ -0,0 +1,479 @@
+---
+title: "Introducing Identity and Access Management (IAM) for OpenFaaS"
+description: "Granularly control access to the OpenFaaS REST API with IAM for OpenFaaS"
+date: 2023-06-20
+categories:
+- sso
+- authorization
+- iam
+- oauth
+author_staff_member: han
+---
+
+## Get started with IAM for OpenFaaS
+
+In this tutorial we are going to walk you through all the steps required to deploy OpenFaaS with Identity and Access Management (IAM) enabled.
+
+In the next sections we are going to:
+
+- Setup ingress and all other prerequisites to deploy OpenFaaS.
+- Configure your identity provider and register it with OpenFaaS.
+- Create roles and policies for fine grained access to the OpenFaaS API.
+- Configure and deploy the OpenFaaS dashboard.
+- Authenticate and manage functions using the faas-cli
+
```

Comment:
Add a brief note on how you configured or set up Keycloak and that we'll be using Keycloak as the IdP for this tutorial.

Reply:
Added instruction for Keycloak